package-res/reportviewer/reportviewer-prompt.js

```diff
@@ -368,12 +368,12 @@ define(['common-ui/util/util', 'pentaho/common/Messages', "dijit/registry", 'com
         if (blocker) {
           messageBox.setButtons([]);
         } else {
-          var closeFunc = function() {
-            if (this.panel) {
-              this.panel.dashboard.hideProgressIndicator();
+          var closeFunc = (function() {
+            if(this.panel) {
```

Comment:
Need try to remove this if statement - we should avoid using prompt panel object directly.